src/Tizen.Security.WebAuthn/Tizen.Security.WebAuthn/PubkeyCredCreationOptions.cs

```diff
@@ -0,0 +1,153 @@
+/*
+ *  Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+
+using System.Collections.Generic;
+using static Tizen.Security.WebAuthn.ErrorFactory;
+
+namespace Tizen.Security.WebAuthn
+{
+    /// <summary>
+    /// Make credential options
+    /// </summary>
+    /// <remarks>
+    /// Refer to the following W3C specification for more information.
+    /// https://www.w3.org/TR/webauthn-3/#dictdef-publickeycredentialcreationoptions
+    /// </remarks>
+    /// <since_tizen> 9 </since_tizen>
+    public class PubkeyCredCreationOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PubkeyCredCreationOptions"/> class.
+        /// </summary>
+        /// /// <since_tizen> 9 </since_tizen>
+        /// <param name="rp">Contains a name and an identifier for the Relying Party responsible for the request</param>
```

Comment:
that's a lot of parameters, if they are optional they maybe specify default values in the method signature so the user will be able to skip those that are not used,  
see https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments?redirectedfrom=MSDN#optional-arguments

Reply:
Done. I made the same change to PubkeyCredRequestOptions.